ote/src/cljs/ote/app/controller/route/route_wizard.cljs

```diff
@@ -525,18 +525,21 @@
                                      time/minutes-from-midnight
                                      (- start-time)
                                      (+ new-start-time)
-                                     time/minutes-from-midnight->time))
-          update-times-from-new-start
-          #(-> %
-               (update ::transit/arrival-time time-from-new-start)
-               (update ::transit/departure-time time-from-new-start))]
+                                     time/minutes-from-midnight->time
+                                     time/time->interval))
+          stop-time-from-template #(-> %
+                                       (cond->
+                                         (some? (::transit/arrival-time %)) (update ::transit/arrival-time time-from-new-start)
+                                         (some? (::transit/departure-time %)) (update ::transit/departure-time time-from-new-start))
+                                       (dissoc ::transit/id ; Remove primary key(s) to avoid updating existing record
```

Comment:
should this be ::transit/route-id?

Reply:
It should be `:transit-stop-time/stop-time-id` which is there, but for some reason I left ::transit/id as well just to be on the safe side for future-proofness. Because failing to remove the ids causes a bug which is not easy to spot.